### DISCUSSION TITLE
Running DevToys 2.0 on Fedora misses Tools

### DISCUSSION THREAD

**Discussion by badcel** [DevToys-app/DevToys#1149] [Q&A] [answered]
Hi,

I tried to run DevToys 2.0 on Fedora 40 and got it running. But it looks like there are no tools available:

![Bildschirmfoto vom 2024-05-06 22-05-45](https://github.com/DevToys-app/DevToys/assets/1218031/bab8a892-4c12-4f23-aa43-d4364a6af31e)

Did I miss some setup step? It looks like there is a [separate repository](https://github.com/DevToys-app/DevToys.Tools) for the tools but the [contributing.md](https://github.com/DevToys-app/DevToys/blob/main/CONTRIBUTING.md) file does not mention it. I wonder how to get some tools running inside DevToys?

**Comment by veler** [comment, marked as answer]
Hello,
Good catch, we should update the Contributing.
As you found, the default tools are in a separated repository. All you need to do is following instructions on this (in progress) documentation, clone the repository and press F5 in VS Code or Rider.

https://github.com/DevToys-app/Documentation/blob/main/articles/extension-development/getting-started/setup.md

**Reply by veler** [reply]
Hello again. I updated the documentation :)

**Reply by badcel** [reply]
Thank you I got it working :-)
